app/helpers/places/finish_line_helper.rb

```diff
@@ -2,24 +2,42 @@ module Places
   module FinishLineHelper
     HOST = 'https://maps.googleapis.com/maps/api/staticmap?'.freeze
 
-    def finish_line_static_map_path(finish_line)
+    def finish_line_static_map(finish_line)
       center = finish_line.center
       options = {
         size: '600x400',
         center: "#{center[:latitude]} #{center[:longitude]}",
         zoom: 15,
         maptype: 'satellite',
-        path: finish_line_static_polyline(finish_line),
         key: maps_api_key
       }
-      HOST + options.to_param
+
+      src = [HOST + options.to_param,
+             finish_line_static_polyline(finish_line),
+             center_line_static_polyline(finish_line, center)].join('&')
+
+      image_tag(src, class: 'img-responsive', alt: finish_line.name)
+    end
+
+    def parameterized_coordinates(coordinates, color, weight)
```

Comment:
So if we move `weight` to constant - no need to have it as a param

Reply:
let's call the method `static_polyline_path`

Reply:
also, lets rearrange arguments and use splat operator for coordinates like:
`static_polyline_path(color, *coordinates)`